nixos/modules/misc/ids.nix

```diff
@@ -278,6 +278,7 @@
       postgrey = 258;
       hound = 259;
       leaps = 260;
+      netdata = 261;
```

Comment:
Looks to me like the data generated by this service is highly machine specific and that ownership will be eventually consistent, given the `preStart` script. Please consider dynamic uid allocation.